app/models/github_repository.rb

```diff
@@ -125,6 +164,10 @@ def tree_url_for_sha(sha)
     html_url + "/tree/" + sha
   end
 
+  def present?(**options)
```

Comment:
Can we move this method back to where it was? It'll simplify the dif a little bit :)